executor/executor-coroutine-android/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+apply from: "$rootDir/gradle/scripts/dependencies.gradle"
+
+android {
+    compileSdkVersion versions.compileSdk
+
+    defaultConfig {
+        minSdkVersion versions.minSdk
+        targetSdkVersion versions.targetSdk
+        versionCode versions.bootsCode
+        versionName versions.boots
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    sourceSets {
+        main.java.srcDirs += "src/main/kotlin"
+        test.java.srcDirs += "src/test/kotlin"
+    }
+
+}
+
+dependencies {
+    implementation project(':core')
+    api project(':executor:executor-coroutine')
```

Comment:
Are you sure that `api` is fine? In this case if user will import only `core` and `executor-coroutine-android` is probably won't work

Reply:
My bad here. Revised gradle docs and you're more that right to use `api` here. 👍